node/silkworm/stagedsync/util.cpp

```diff
@@ -44,7 +44,11 @@ std::pair<Bytes, Bytes> convert_to_db_format(const ByteView& key, const ByteView
 }
 
 void truncate_table_from(mdbx::cursor& table, Bytes& starting_key, bool reverse) {
-    if (table.lower_bound(db::to_slice(starting_key), false)) {
+    auto current{table.lower_bound(db::to_slice(starting_key), false)};
```

Comment:
Please document what truncate table is supposed to do in the reverse case. When `reverse` is false, it deletes all entries whose `key>= starting_key`, correct? What is it supposed to do when `reverse` is true?

Reply:
@Giulio2002 documentation for truncate_table_from is still missing. From the code I take it that it  deletes all entries whose `key >= starting_key` when reverse is false, and all entries whose `key < starting_key` when reverse is true. Is that the intention? It should be documented.

Reply:
so the thing is this, there is no such thing as an upper_bound function, so my solution for whenever we need to reverse is we take key+1 and go to previous element so that we are at the upper bound

Reply:
forgot to address this sorry